src/ApiService/Tests/RequestsTests.cs

```diff
@@ -35,6 +35,7 @@ public class RequestsTests {
         var deserialized = (T?)_serializer.Deserialize(stream, typeof(T), CancellationToken.None);
         var reserialized = _serializer.Serialize(deserialized);
         var result = Encoding.UTF8.GetString(reserialized);
+        result = result.Replace(System.Environment.NewLine, "\n");
```

Comment:
this line is making the roundtrip fail for me 

```
Xunit.Sdk.EqualException
Assert.Equal() Failure
           ↓ (pos 1)
Expected: {\r\n  "event": {\r\n    "state_update": {\r\n  ···
Actual:   {\n  "event": {\n    "state_update": {\n     ···
           ↑ (pos 1)
   at Tests.RequestsTests.AssertRoundtrips[T](String json) in C:\work\onefuzz\src\ApiService\Tests\RequestsTests.cs:line 39
   at Tests.RequestsTests.NodeEvent_StateUpdate_Busy() in C:\work\onefuzz\src\ApiService\Tests\RequestsTests.cs:line 165
```

Reply:
ha, for me it's the other way around.

if you remove the line, the test passes for you ?

Reply:
yes it works when removing the line